atc/atccmd/command.go

```diff
@@ -148,6 +148,9 @@ type RunCommand struct {
 	DefaultBuildLogsToRetain uint64 `long:"default-build-logs-to-retain" description:"Default build logs to retain, 0 means all"`
 	MaxBuildLogsToRetain     uint64 `long:"max-build-logs-to-retain" description:"Maximum build logs to retain, 0 means not specified. Will override values configured in jobs"`
 
+	DefaultBuildLogsDaysToRetain uint64 `long:"default-build-logs-days-to-retain" description:"Default days to retain build logs. 0 means unlimited"`
```

Comment:
The name here feels odd - how about `default-days-to-retain-build-logs` and `max-days-to-retain-build-logs`?